src/krem.js

```diff
@@ -0,0 +1,7 @@
+export function krem(strings, ...args) {
```

Comment:
So if I understand correctly, the `krem` template string *must* be called with interpolated values and doesn't work with an actual literal?

So this works:
```js
krem`${namespace}${id}${styles}`;
```

But this doesn't:
```js
krem`krem3.hi{}`
```

Am I understanding correctly? How do you plan on getting kremling-loader to create the `namespace`, `id`, and `styles` variables? That seems like a pretty extensive compilation process to add variables to the scope.

Here's an alternate API for krem that might be easier to implement inside of kremling-loader.

```js
function krem(strings) {
  const evaluated = 
  const [namespace, id, styles] = strings[0].split('|')
  return {namespace, id, styles}
}

krem`my-namespace|3|.hi {}`
```

I'm pretty sure that `|` is safe to use as a delimiter in css strings because i think it's an invalid character. We could also use `&` which I *know* is invalid :)

All of this ^ is just ideas -- i haven't even taken a look at the kremling-loader PR, so maybe the implementation there isn't as complicated or hard as I'm thinking.

Reply:
interesting - I actually like that better...

 cause if we use pipes, we could gracefully fallback to a string